.github/workflows/pr.yaml

```diff
@@ -47,29 +47,7 @@ jobs:
         run: |
           composer install --no-interaction --no-progress
           composer normalize composer.json --dry-run
-
-  config-check:
-    name: Check that config is up to date
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@master
-
-      - name: Install site
-        run: |
-          docker network create frontend
-          docker compose pull
-          docker compose up --detach
-
-          # Important: Use --no-interaction to make https://getcomposer.org/doc/06-config.md#discard-changes have effect.
-          docker compose exec --user root phpfpm composer install --no-interaction
-
-          # Install the site
-          docker compose exec --user root phpfpm vendor/bin/drush site:install --existing-config --yes
-
-      - name: Export config
-        run: docker compose exec --user root phpfpm vendor/bin/drush config:export --yes
-      - name: Check for changes in config
-        run: git diff --diff-filter=ACMRT --exit-code config/
```

Comment:
Why has this been removed?

Reply:
It doesn't correctly report config changes